services/app/app/templates/portal/index.hbs

```diff
@@ -27,6 +27,12 @@
         Modify directory categories
       {{/link-to}}
     {{/if}}
+    {{#if onlineDirectoryEnabled}}
```

Comment:
I'd prefer if we used `directorySections` as the prefix for everything instead of `onlineDirectory`.